### ISSUE TITLE
Latest rswag-ui does not work

### ISSUE THREAD

**Issue by indocomsoft** [rswag/rswag#124]
One needs to manually go to the rswag-ui directory and run `yarn install` before it would work.

**Comment by domaindrivendev** [comment]
If you pull it down as a gem (2.0.0), it should work without any additional step. I assume you mean it doesn't work when running locally from source?

If so, this is by design because rather than copying the swagger-ui files into this repo, they are instead pulled down via npm at build-time. Sounds like you got this to work with yarn - you can also run ci/build.sh from this repo.

**Comment by pedantic-git** [comment]
@domaindrivendev: This is happening to me in the gem version of rswag-ui too: I'm not seeing the assets being installed and they all 404 when the page loads.

I can reproduce it just by doing:

```
rails new rswag-test
cd rswag-test
vim Gemfile # add 'rswag-ui'
bundle
rails g rswag:ui:install
rails s
```

If you visit http://localhost:3000/api-docs you'll get the blank screen with loads of 404 assets.

**Comment by pedantic-git** [comment]
Additional to this: looking at the installed gem:

```
rails@384ad31499e5:/bundle/gems/rswag-ui-2.0.0$ ls
MIT-LICENSE  Rakefile  lib
```

The gemspec says to include node_modules but they're not included - is it possible the build of the version on rubygems happened in the wrong order?

**Comment by indocomsoft** [comment]
@domaindrivendev is version 2.0.0 from latest master? The rswag-ui 2.0.0 pulled by bundler on my machine looks like the way it is before the rewrite (without package.json etc.)

What I meant when I created this issue is when specifying on Gemfile to pull rswag from this git repo, everything would not work without me manually going to the rswag directory and running `yarn install` there. Perhaps this could be made into a rake task to be run after installing rswag.

**Comment by pedantic-git** [comment]
@indocomsoft The [gemspec](https://github.com/domaindrivendev/rswag/blob/master/rswag-ui/rswag-ui.gemspec) for rswag-ui isn't meant to include package.json but it is meant to include the node_modules (although no build failure if it's absent), so I think the problem is that whoever built the gem that was uploaded to rubygems.org didn't run `npm` before building it. Hopefully @domaindrivendev can confirm.

**Comment by domaindrivendev** [comment]
Ok sounds like an issue with CI/CD although I thought I’d pulled it down and checked. Will take a look later today. Thanks for your patience

**Comment by domaindrivendev** [comment]
Had a small amount of time to today but was unable to resolve the faulty CI/CD process. If anyone has any Travis knowledge and/or spare cycles, some help would be appreciated. I suspect it's something related to the use of "Build stages", which feel overly complicated at this point. When I build the gem locally it contains the folder, when I do it as part of the publish from Travis it doesn't get included.

**Comment by indocomsoft** [comment]
@domaindrivendev perhaps try `skip_cleanup: true` as in https://docs.travis-ci.com/user/deployment/rubygems/#Releasing-build-artifacts? I suspect travis might clean up the `node_modules` directory:
> After your tests ran and before the release, Travis CI will clean up any additional files and changes you made.

**Comment by indocomsoft** [comment]
@domaindrivendev yup, I just tested, it should work. I made PR #125 to fix this issue.

**Comment by domaindrivendev** [comment]
Should be fixed as of version 2.0.3
